ang/civicase/DashboardTab.html

```diff
@@ -35,9 +38,9 @@
           <panel-query-results>
             <div ng-repeat="activity in results"
               case-activity-card="activity"
-              refresh-callback="loadData"
+              refresh-callback="customData.cardRefresh"
               mode="long"
-              refresh-on-checkbox-toggle="false"
+              refresh-on-checkbox-toggle="true"
```

Comment:
isn't this the default behaviour? I think this does not need to be specified.

Reply:
I can't see a default value being assigned to this param if not passed to the directive, see https://github.com/compucorp/uk.co.compucorp.civicase/blob/develop/ang/civicase/ActivityCard.js#L25 and https://github.com/compucorp/uk.co.compucorp.civicase/blob/develop/ang/civicase/ActivityCard.js#L67